extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -237,7 +237,7 @@ export class AmpStoryInteractive extends AMP.BaseElement {
   }
 
   /** @override */
-  buildCallback(concreteCSS = '') {
+  buildCallback(concreteCSS = '', extraCSS = '') {
```

Comment:
Might be better off to have the concreteCSS be the sum of interactive.css + img-quiz.css so we don't change the abstract component to achieve that behavior. Then, you just pass one string here